<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MixControl</class>
 <widget class="QGroupBox" name="MixControl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GroupBox</string>
  </property>
  <property name="title">
   <string>Composition</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>133</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1">
    <widget class="QToolButton" name="f_edit">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" rowspan="5">
    <widget class="QTreeView" name="treeView"/>
   </item>
   <item row="1" column="1">
    <widget class="QToolButton" name="f_remove">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QToolButton" name="f_add">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
